frontend/src/upload.js

```diff
@@ -345,6 +358,9 @@ $(document).ready(function() {
       class: 'icon-copy',
       'data-l10n-id': 'copyUrlHover'
     });
+    if (!allowedCopy()) {
+      $copyIcon.addClass('disabled');
```

Comment:
since 'disabled' is a really common, can we add a 'disabled' attribute instead? Then check the css with [disabled]
we already have a rule for `#copy-btn:disabled` that can be reused.